app/models/skin.rb

```diff
@@ -426,7 +443,7 @@ def self.load_site_css
           skin.official = true
           File.open(version_dir + 'preview.png', 'rb') {|preview_file| skin.icon = preview_file}
           skin.save!
-
+          Rails.cache.increment('skins_generation/'+skin.type.nil? ? ("site_skin") : skin.id.to_s )
```

Comment:
Surrounding space missing for operator `+`.<br>Space inside parentheses detected.